resources/js/pages/settings/roles.tsx

```
@@ -0,0 +1,227 @@
+    roles: {
+        id: string;
+        name: string;
+    }[];
+    permissions: {
+        id: string;
+        name: string;
+        model: string;
+    }[];
+}
+
+interface User {
```

ESLint

[error] 'User' is defined but never used.

(@typescript-eslint/no-unused-vars)